Exercise_D.js

```diff
@@ -14,7 +14,14 @@
 Use a combination of conditionals and loops to create your answer for this question. */
 
 function multiplyNumber(number) {
-  // CODE HERE
-}
+  if(number <= 0 || isNaN(number)) {
+    return "ERROR"
+  } else if (number >= 1000000) {
+    return number
+  } else  {
+    while (number < 1000000) {number = number * 10};
+    return number
+  }
+} 

```

Comment:
Nice work, the only improvement that I can see quickly can be made on line 22.
while (number < 1000000) { number *= 10};